Cartfile

```diff
@@ -1 +1 @@
-github "hyperoslo/Cache" "master"
+github "hyperoslo/Cache" ~> 3.0
```

Comment:
I just think that they should remove this `~>` and be explicit to use `3.x` instead. Remove all the custom operators